src/main/modules/oidc/index.ts

```diff
@@ -60,33 +59,30 @@ export class OidcMiddleware {
     });
 
     app.use(
-      errorHandler(async (req: RequestWithScope, res: Response, next: NextFunction) => {
-        if (req.session.user?.id_token) {
+      errorHandler(async (req: AppRequest, res: Response, next: NextFunction) => {
+        if (req.session.user) {
           res.locals.isLoggedIn = true;
-
-          const user = req.session.user;
-          req.scope = req.app.locals.container.createScope();
-          req.scope?.register({
-            axios: asValue(
-              Axios.create({
-                baseURL: config.get('services.cos.baseURL'),
-                headers: {
-                  Authorization: 'Bearer ' + user.access_token,
-                  IdToken: user.id_token,
-                },
-              })
-            ),
-            api: asClass(CaseApi),
-          });
+          req.locals.api = new CaseApi(
```

Comment:
im starting to think, is it best to create an api controller every time the user sends a request?

its not a big deal its just made me think if we should check before create maybe?